SQL/Release_patches/18.0_19.0_upgade.sql

```diff
@@ -0,0 +1,10 @@
+/*! remove Radiological Reviews from menu */
+DROP TABLE IF EXISTS `final_radiological_review_history`;
+DROP TABLE IF EXISTS `final_radiological_review`;
+
+DELETE FROM LorisMenu where Label = "Radiological Reviews";
+update LorisMenu set OrderNumber = (OrderNumber-1) where Parent = 3;
+
+delete from permissions where code = "edit_final_radiological_review";
```

Comment:
This will probably make the integrity constrain FK_user_perm_rel_2 on user_perm_rel complain a little.